src/Orleans/GrainDirectory/MultiClusterRegistrationStrategy.cs

```diff
@@ -8,7 +9,7 @@ namespace Orleans.GrainDirectory
     /// Strategy objects are used as keys to select the proper registrar.
     /// </summary>
     [Serializable]
-    internal abstract class MultiClusterRegistrationStrategy
+    public abstract class MultiClusterRegistrationStrategy
```

Comment:
why is this declared to public ? when all the methods is internal and private. 
just to trying to understand